ljh/ljh.go

```diff
@@ -309,12 +309,14 @@ func (w *Writer3) WriteRecord(firstRisingSample int32, rowcount int64, timestamp
 
 // Flush flushes buffered data to disk
 func (w Writer3) Flush() {
-	w.writer.Flush()
+	if w.writer != nil {
+		w.writer.Flush()
+	}
```

Comment:
Good. This was a problem with the LJH 2 writer, too, right?

Reply:
Yes, I'm surprised it doesn't seem to be a problem with `w.file` as well.